libbpfgo.go

```diff
@@ -328,6 +328,41 @@ func NewModuleFromFile(bpfObjPath string) (*Module, error) {
 	})
 }
 
+// LibbpfStrictMode is an enum as defined in https://elixir.bootlin.com/linux/v5.17.5/source/tools/lib/bpf/libbpf_legacy.h#L23
```

Comment:
Let's link to libbpf explicitly, instead of the linux source code which isn't typically in sync (libbpf changes make their way down into mainline)

https://github.com/libbpf/libbpf/blob/2cd2d03f63242c048a896179398c68d2dbefe3d6/src/libbpf_legacy.h#L23

Reply:
Oh, good call, thought it was the other way around for some reason!